packages/kit/src/core/build/index.js

```diff
@@ -314,6 +314,10 @@ async function build_server(
 					get_component_path: id => ${s(`${config.kit.paths.assets}/${config.kit.appDir}/`)} + entry_lookup[id],
 					get_stack: error => String(error), // for security
 					handle_error: error => {
+						// frame is populated by Svelte in its CompileError and is a Rollup/Vite convention
+						if (error.frame) {
+							console.error(error.frame);
+						}
```

Comment:
Does this not need the type hint provided in the `dev` equivalent?

Reply:
No idea. None of the tests / examples do type checking so there's not an easy way to check and we're constantly braking things. I went ahead and added it